scripts/envDFL.sh

```diff
@@ -60,6 +60,34 @@ export_var_env_force() {
   export $name="$value"
 }
 
+export_var_env_force_dynawo() {
+  local var="$@"
+  local name=${var%%=*}
+  local value="${var#*=}"
+
+  if ! `expr $name : "DYNAWO_.*" > /dev/null`; then
+    error_exit "You must export variables with DYNAWO prefix for $name."
+  fi
+
+  if eval "[ \"\$$name\" ]"; then
+    unset $name
+    export $name="$value"
+    return
+  fi
+
+  if [ "$value" = UNDEFINED ]; then
+    error_exit "You must define the value of $name"
+  fi
+  export $name="$value"
+}
+
+export_var_env_dynawo() {
```

Comment:
why adding a method with the exact same objectives as  export_var_env_force_dynawo?

Reply:
Renamed to export_var_env_dynawo_with_default